tools/data/add_transcripts_from_gff3_to_annotations.pl

```diff
@@ -142,6 +144,7 @@ sub print_usage {
     [--success_log|-l <success log file>]
     [--error_log|-L <error log file>]
     [--skip|-s <skip id file>]
+    [--test|-x]
```

Comment:
The idea of `--test` was for it to serve as an aid for diagnostics when parsing a GFF3 using this script.